evap/staff/tests/test_views.py

```diff
@@ -9,10 +9,122 @@
 import xlrd
 
 from evap.evaluation.models import Semester, UserProfile, Course, CourseType, TextAnswer, Contribution, \
-                                   Questionnaire, Question, EmailTemplate, Degree
+                                   Questionnaire, Question, EmailTemplate, Degree, FaqSection, FaqQuestion
 from evap.evaluation.tests.tools import FuzzyInt, WebTest, ViewTest
 
 
+class TestStaffIndexView(ViewTest):
+    test_users = ['staff']
+    url = '/staff/'
+
+    def setUp(self):
+        mommy.make(UserProfile, username='staff', groups=[Group.objects.get(name='Staff')])
+
+
+class TestStaffFAQView(ViewTest):
+    url = '/staff/faq/'
+    test_users = ['staff']
+
+    def setUp(self):
+        mommy.make(UserProfile, username='staff', groups=[Group.objects.get(name='Staff')])
+
+
+class TestStaffFAQEditView(ViewTest):
+    url = '/staff/faq/1'
+    test_users = ['staff']
+
+    def setUp(self):
+        mommy.make(UserProfile, username='staff', groups=[Group.objects.get(name='Staff')])
+        section = mommy.make(FaqSection)
+        mommy.make(FaqQuestion, section=section)
+
+
+class TestSemesterCreateView(ViewTest):
+    url = '/staff/semester/create'
+    test_users = ['staff']
+
+    @classmethod
+    def setUpTestData(cls):
+        mommy.make(UserProfile, username='staff', groups=[Group.objects.get(name='Staff')])
+
+    def test_create(self):
+        name_de = 'name_de'
+        name_en = 'name_en'
+
+        response = self.app.get(self.url, user='staff')
+        form = response.forms['semester-form']
+        form['name_de'] = name_de
+        form['name_en'] = name_en
+        form.submit()
+
+        self.assertTrue(Semester.objects.get(name_de=name_de, name_en=name_en))
```

Comment:
done. And a lot of tests in test_views actually test forms. We should find a consistent way to deal with this.